pysat/tests/test_meta.py

```diff
@@ -1067,6 +1162,8 @@ def test_contains_case_insensitive_w_ho(self):
         assert ('NEW21' not in self.meta)
 
     def test_get_variable_name_case_preservation(self):
+        """Test `meta.var_case_name` preserves the case (is case independent?).
```

Comment:
@rstoneback, how about this?
```suggestion
        """Test `meta.var_case_name` preserves the required output case.
```